### DISCUSSION TITLE
SelectButton - How to change background

### DISCUSSION THREAD

**Discussion by gambithunt** [primefaces/community#2990] [PrimeVue]
Hi

When looking at the docs on SelectButton - Theming, I dont see the usual options in the design tokens to change the background or any other details for that matter, it seems to have very few options - 

```
selectbutton.border.radius
selectbutton.invalid.border.color
```

Please could you let me know how to go about customizing this component.

**Comment by j0rgedev** [comment]
Hello @gambithunt. I realize this isn't well-documented, but the SelectButton component actually uses the ToggleButton under the hood. This is why only a few options appear in the theming tab for SelectButton.

For convenience, here’s an example of a custom preset using the ToggleButton component. Hope this helps!

```js
const MyPreset = definePreset(Aura, {
  components: {
    togglebutton: {
      colorScheme: {
        light: {
          root: {
            background: 'red',
          },
        },
      },
    },
  },
});
```
Read more about [Component Customization](https://primevue.org/theming/styled/#component)

**Comment by gambithunt** [comment]
Thanks very much, it works
